app/_locales/es/messages.json

```diff
@@ -929,5 +929,579 @@
   },
   "yourSigRequested": {
     "message": "Tu firma ya fue solicitada"
+  },
+  "contractInteraction": {
+    "message": "Interacción del Contrato"
+  },
+  "providerRequest": {
+    "message": "$1 quisiera conectar con tu cuenta"
+  },
+  "acceleratingATransaction": {
+    "message": "* Agilizar a una transacción al usar un precio de gas más alto aumenta las probabilidades que que la red lo procese más rápidamente, pero eso no siempre está garantizado."
+  },
+  "accessingYourCamera": {
+    "message": "Accediendo a tu cámara..."
+  },
+  "accountOptions": {
+    "message": "Opciones de la Cuenta"
+  },
+  "accountSelectionRequired": {
+    "message": "Hay que elegir una cuenta!"
+  },
+  "activityLog": {
+    "message": "registro de movimientos"
```

Comment:
"registro de actividad"

Reply:
My thinking here was this:
-Registro de actividad: log of the user's activities using MetaMask
-Registro de movimientos: log of the user's (monetary) transactions using MetaMask

I'm not sure which one is meant, here.

Reply:
That said, I don't want to hold up the entire PR for this. 'actividades' is probably understood and general enough for both meanings. I'll change it to that.